src/tests/JIT/HardwareIntrinsics/Arm/AdvSimd.Arm64/Abs.Vector128.Double.cs

```diff
@@ -13,12 +13,14 @@
 using System.Runtime.InteropServices;
 using System.Runtime.Intrinsics;
 using System.Runtime.Intrinsics.Arm;
+using Xunit;
 
-namespace JIT.HardwareIntrinsics.Arm
+namespace JIT.HardwareIntrinsics.Arm._AdvSimd.Arm64
```

Comment:
Just wondering, why the need for the underscore here?

Reply:
Its been quite a while, but I think there was a conflict between a namespace name and class name, and I stuffed in an underscore on the namespace to shut it all up.